qcodes/instrument_drivers/Keysight/Keysight_N5230A.py

```diff
@@ -0,0 +1,160 @@
+                           get_cmd='OUTP?',
+                           set_cmd='OUTP {}',
+                           val_mapping={'on': 1, 'off': 0})
+
+        self.add_parameter('electrical_delay',
+                           label='Electrical Delay',
+                           get_cmd=':CALC:CORR:EDEL:TIME?',
+                           set_cmd=':CALC:CORR:EDEL:TIME {:.16f}',
+                           get_parser=float,
+                           unit='s')
+
+        self.add_parameter('bandwidth',
+                           label='Bandwidth',
+                           get_cmd=':SENS:BAND?',
+                           set_cmd=':SENS:BAND {:.16f}',
+                           get_parser=float,
+                           unit='Hz')
+
+        self.add_parameter('center_frequency',
+                           label='Center Frequency',
+                           get_cmd=':SENS:FREQ:CENT?',
+                           set_cmd=':SENS:FREQ:CENT {}',
+                           get_parser=float,
+                           unit='Hz')
+
+        self.add_parameter('start_frequency',
+                           label='Start Frequency',
+                           get_cmd=':SENS:FREQ:STAR?',
+                           set_cmd=':SENS:FREQ:STAR {}',
+                           get_parser=float,
+                           unit='Hz')
+
+        self.add_parameter('stop_frequency',
+                           label='Stop Frequency',
+                           get_cmd=':SENS:FREQ:STOP?',
+                           set_cmd=':SENS:FREQ:STOP {}',
+                           get_parser=float,
+                           unit='Hz')
+
+        self.add_parameter('span',
+                           label='Span',
+                           get_cmd=':SENS:FREQ:SPAN?',
+                           set_cmd=':SENS:FREQ:SPAN {}',
+                           get_parser=float,
+                           unit='Hz')
+
+        self.add_parameter('s_parameter',
+                           label='S parameter',
+                           docstring='Set the measured S parameter, allowed values are "S11", "S12", "S21", and "S22"',
+                           get_cmd=':CALC:PAR:CAT?',
+                           set_cmd=':CALC:PAR:MOD  {}',
+                           get_parser=lambda x: x[1:-1].split(',')[1])
+
+        self.add_parameter('num_points',
+                           label='Number of points',
+                           get_cmd=':SENS:SWE:POIN?',
+                           set_cmd=':SENS:SWE:POIN  {}',
+                           get_parser=int)
+
+        self.add_parameter('num_averages',
+                           label='Number of Averages of a sweep',
+                           get_cmd=':SENS1:AVER:COUN?',
+                           set_cmd=':SENS1:AVER:COUN {}',
+                           get_parser=int)
+
+        self.add_parameter('averaging',
+                           label='Enable Averaging',
```

Comment:
"Enable averaging" -> "Enable/disable averaging" (since disabling also happens through this parameter)